undertow/WFLY-12473_obfuscate-session-route.adoc

```diff
@@ -0,0 +1,79 @@
+= WFLY-12473 Obfuscate session-route
+:author:            Flavia Rainone
+:email:             frainone@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          undertow,instance-id,obfuscate-session-route
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This feature is to be able to obfuscate the value of UndertowService.instanceId when routing. Routes are used as
+part of generated session ids when a remote invocation (such as EJB, naming or serving a wildfly-services address)
+is handled by HTTP Invoker. This data is sent across connections as part of the request and response JSESSION_ID
+cookie.
+EAP6 Web Server would similarly allow obfuscation of instance-id at routing by allowing a generated
+UUID to be used as the instance-id.
+This UUID was the default value used whenever instance-id configuration was absent, while Undertow default value
```

Comment:
@fl4via do you mean `, while Undertow default value with EAP 7, for instance-id is ${jboss.node.name}.` ?

Reply:
Yes, I'll update